.github/workflows/ci.yml

```
@@ -1,7 +1,19 @@
 name: CI
-on: pull_request
+on: [push, pull_request]
 jobs:
-  approve:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: actions/setup-node@v2.1.5
+        with:
+          node-version: '14.x'
+      - run: npm i
```

let's prefer npm ci when there's a lockfile

```suggestion
      - run: npm ci
```